appengine/flexible/multiple_services/api_gateway/gateway/api_gateway.py

```diff
@@ -1,74 +1,44 @@
 import requests
 import sys
-import os
 from flask import Flask
+import services_config
 app = Flask(__name__)
+app.config['SERVICE_MAP'] = services_config.map_services('production')
 
-@app.route('/environment')
+@app.route('/env')
 def get_env():
-    configuration = 'production'
-    return app.config['MODE']
-
-@app.route('/test')
-def test():
-    return "GATEWAY OPERATIONAL"
+    print(app.config['SERVICE_MAP'])
+    return app.config
```

Comment:
returning the entire app.config can be a security issue, can you limit the scope here?